core/domain/stats_jobs_one_off.py

```diff
@@ -390,7 +543,9 @@ def reduce(key, stringified_values):
 
 
 class PartialAnswerValidationAudit(jobs.BaseMapReduceJobManager):
-    """Perform an answer value validation for numeric input answers submitted to
+    """Perform an answer value validation for numeric input.
+
+    Perform an answer value validation for numeric input answers submitted to
```

Comment:
FYI it's OK (in this codebase) if the initial docstring doesn't fit on a single line. No need to duplicate it like this. Ditto below.